basis/io/sockets/secure/secure.factor

```diff
@@ -114,3 +114,5 @@ HOOK: accept-secure-handshake secure-socket-backend ( -- )
     { [ os unix? ] [ "io.sockets.secure.unix" require ] }
     { [ os windows? ] [ "io.sockets.secure.windows" require ] }
 } cond
+
+M: object opt-send-secure-handshake send-secure-handshake ;
```

Comment:
It's supposed to mean that it's an optional compilation dependency. I was considering using the same name :  "send-secure-handshake" but I think it's confusing. But I don't like the opt- prefix either, do you have a better idea ?
(or a different solution would be to stop doing optional compilation dependencies to io.sockets.secure from http.client)